packages/v4/src/content/design-guidelines/usage-and-behavior/lists-and-tables/lists-and-tables.md

```diff
@@ -8,149 +8,201 @@ showTitle: true
 
-![actionable data list](./img/actionable-list.png)
+<img src="./img/actionable-list.png"  alt="Actionable list"  width="1174"/> 
 
 1. **Checkbox:** Enables a user to select a row
 2. **Global actions:** Actions that can be applied to all selected items
 
-#### When to use
-Use actionable data lists when you need to enable a user to select and act on multiple items in the list.
+#### When to use and when not
+Use an actionable data list when:
+* You need to enable a user to select and act on multiple items in the list.
 
-#### When not to use
-Do not use actionable data lists when actions are restricted to a single row or object. In this case, place actions inline within the row.
+Don't use an actionable data list when:
+* Actions are restricted to a single row or object. In this case, place actions inline within the row.
 
 ### Expandable data list
 The expandable list adds an expansion panel to every row to reveal more details about the item.
 
-![expandable data list](./img/expandable-list.png)
-
+<img src="./img/expandable-list.png"  alt="Expandable data list"  width="1174"/> 
 
 1. **Expansion caret:** Toggles the expansion open and closed
 2. **Expansion panel:** A container that is revealed when the expansion is open. It can accept any supported layout to present additional content to the user.
 
 #### When to use
-Use an expandable data list when you have more information than will comfortably fit inside a row, or you want to provide a way for advanced users to access information that is not applicable to all users.
+Use an expandable data list when:
+* You have more information than will comfortably fit inside a row.
+* You want to provide a way for advanced users to access information that is not applicable to all users.
+
+## Tables 
+
+### How to use
+Consider the structure of the data you want to display and organize that information into columns. Columns will typically have column headers. Every row within a table must have a consistent format. If the table row includes actions, they should always be placed in the rightmost column(s).
 
-## Tables
-* **Standard table:** Combines the styling of a list view with behaviors that are commonly associated with a table (sometimes referred to as a hybrid list)
-* **Compact table:** Maximizes the amount of data that can be displayed in a small space
+### Compact and default table
+PatterFly supports two main types of tables:
 
-### Standard table
+1. **[Compact table](#compact-and-default-table)** - when you want to show as much data per page as possible.
+2. **[Default table](#compact-and-default-table)** - when you don’t have to minimize paging.
 
-![standard table](./img/standard-data-table.png)
+See [when to use compact vs. default spacing](#when-to-use-compact-vs-default-spacing) for more information about the styling and usage.
+
+<img src="./img/compact-and-default-table.png"  alt="Compact and default table"  width="857"/> 
+
+### Table elements
```

Comment:
In general we call out more things here than for the Data list. Not sure which approach is better, but we should be consistent.